packages/core/src/util/cell.ts

```diff
@@ -156,8 +161,8 @@ export const defaultMapStateToEnumCellProps = (
   ownProps: OwnPropsOfEnumCell
 ): StatePropsOfEnumCell => {
   const props: StatePropsOfCell = mapStateToCellProps(state, ownProps);
-  const options =
-    ownProps.options !== undefined ? ownProps.options : props.schema.enum;
+  const options: EnumOption[] =
+    ownProps.options !== undefined ? ownProps.options : props.schema.enum.map(enumToEnumOptionMapper) || [enumToEnumOptionMapper(props.schema.const)];
```

Comment:
This will cause runtime errors when `props.schema.enum` is undefined.

Reply:
Formatting is currently off (working on it), so would be great to format this line manually.